packages/video_player/video_player/pubspec.yaml

```diff
@@ -3,7 +3,8 @@ description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android, iOS, and web.
 repository: https://github.com/flutter/plugins/tree/main/packages/video_player/video_player
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+video_player%22
-version: 2.4.7
+version: 2.4.8
```

Comment:
This should be a major version bump since it is adding a feature. (e.g. `2.5.0`). Although the exact value may change by the time the PR is ready to land.